kerastuner/engine/metrics_tracking.py

```diff
@@ -54,6 +56,22 @@ def __eq__(self, other):
         return (other.value == self.value and
                 other.step == self.step)
 
+    def to_proto(self):
+        value = self.value
+        # Must always be saved as a list.
+        if not isinstance(value, list):
```

Comment:
It seems like this should be standardized inside the class rather than only at serialization time (e.g. self.value should always be a list, or in the case perhaps a set since order doesn't matter). Would this cause any problem?

Reply:
My thinking was just that this way users don't have to be aware that MetricObservations can have a list of values unless they pass multiple values to it, wdyt?

Alternatively if we still want that behavior we could have a flag on the proto that saves whether or not `values` was a list

Reply:
Discussed offline, will make MetricObservation value always a list